admin/tools/docker/configured/Dockerfile.tpl

```diff
@@ -0,0 +1,15 @@
+FROM {{DOCKER_IMAGE_OPT}}
+MAINTAINER Fabrice Jammes <fabrice.jammes@in2p3.fr>
+
+WORKDIR /qserv
+
+USER qserv
+
+RUN bash -c ". /qserv/stack/loadLSST.bash && setup qserv -t qserv-dev && /qserv/scripts/configure.sh {{NODE_TYPE_OPT}} {{MASTER_FQDN_OPT}}"
```

Comment:
I don't suppose it would be possible to use `docker run --env=...` in place of having to hard code the hostnames?